Products/Zuul/catalog/model_catalog.py

```diff
@@ -263,49 +261,66 @@ def add_model_update(self, object_update):
         if idxs: # make sure mandatory idxs are always sent
             idxs.update( MANDATORY_FIELDS ) # Mandatory fields
 
-        model_update = IndexUpdate(object_update.obj, op=op , idxs=idxs, uid=uid)
-        self.pending_updates[object_update.uid] = model_update
-        del object_update # Make sure we dont keep references to the object
```

Comment:
should we still do this?

Reply:
I think it was redundant and the python garbage collector will do its job